os_migrate/playbooks/delete_conversion_hosts.yml

```diff
@@ -40,9 +40,7 @@
         os_migrate_conversion_manage_network: "{{ os_migrate_src_conversion_manage_network|default(true) }}"
         os_migrate_conversion_manage_fip: "{{ os_migrate_src_conversion_manage_fip|default(true) }}"
         os_migrate_conversion_delete_fip: "{{ os_migrate_src_conversion_host_delete_fip|default(true) }}"
-        os_migrate_conversion_auth: "{{ os_migrate_src_auth }}"
-        os_migrate_conversion_auth_type: "{{ os_migrate_src_auth_type|default(omit) }}"
-        os_migrate_conversion_region_name: "{{ os_migrate_src_region_name|default(omit) }}"
+        cloud: src
```

Comment:
This (and the similar spots) is why the CI is failing. Instead of using `cloud` here directly as on the modules, let's define `os_migrate_conversion_cloud: src` here and update the spots that make use of the `os_migrate_conversion_auth` variable to use `os_migrate_conversion_cloud` variable instead:

https://github.com/os-migrate/os-migrate/blob/e0359b14779e12c70dd8ebbacf09a47fdd020d5b/os_migrate/roles/conversion_host/tasks/conv_host_inventory.yml#L4-L6

So we'd put there:

```
cloud: "{{ os_migrate_conversion_cloud }}"
```
This is very similar to the replacement we've been doing, but instead of hardcoding a src/dst value, we'll be going through a variable, so that we can include the same tasks once for src context, and once for dst context.